docs/ci-cd.md

```diff
@@ -1,3 +1,34 @@
 # CI/CD
 
-TODO
+### GitLab CI/CD
+
+Set the following variables in GitLab project:
+
+- `SSH_PRIVATE_KEY`: Content of `~/.ssh/known_hosts` file. 
+The public SSH keys for a host may be obtained using the utility `ssh-keyscan`. 
+For example: `ssh-keyscan deployer.org`.
+- `SSH_KNOW_HOSTS`: Private key for connecting to remote hosts. 
+To generate private key: `ssh-keygen -t ed25519 -C 'gitlab@deployer.org'`.
+
+Create .gitlab-ci.yml file with following content:
+
+```yml
+stages:
+  - deploy
+
+deploy:
+  stage: deploy
+  image:
+    name: debreczeniandras/deployerphp:7-beta
+    entrypoint: [""]
+  before_script:
+    - mkdir -p ~/.ssh
+    - eval $(ssh-agent -s)
```

Comment:
Maybe better to add note what this method is using forward agent with same private key to access host and clone repo? 

Also will be cool to mention deploy keys.

Reply:
In our workflow we don't clone the repo since GitLab CI job already clone the repo.
Instead we use `rsync` recipe to upload the code to the host.
I would try to add in the docs, WDYT?

Reply:
👍🏻 Yes, actually this is the section there I'd like to describe on how to change deploy:update_code for CI/CD.

Reply:
Done :wink: